docs/bib/media.bib

```diff
@@ -1,3 +1,13 @@
+@article{CERN-COURIER-61-3_May_2021,
+  title   = {{LHC reinterpreters think long-term}},
+  author  = {Sabine Kraml},
+  journal = {CERN Courier Volume 61, Number 3, May/June 2021},
+  year    = {2021},
+  month   = {May},
+  url     = {https://cerncourier.com/wp-content/uploads/2021/04/CERNCourier2021MayJun-digitaledition.pdf},
+  note    = {https://cds.cern.ch/record/2765233}
```

Comment:
I opted to put the CERN Courier link in as the "url" field and then add the permanent CDS archive URL as a note. If this looks weird in the render (c.f. RTD preview) or if there are suggestions we can revise this.